style.css

```diff
@@ -0,0 +1,68 @@
+* {
+  box-sizing: border-box;
+  color: purple;
+  font: arial;
+  margin: 0, 5em;
+  padding: 0%;
+}
+/* header and nav */
+
+header {
+  background-color: rgb(255, 237, 255);
+  text-align: center;
+  color: rgb(52, 0, 59);
+}
+nav {
+  margin: 1em;
```

Comment:
I would use "rem" as a unit most of the time. Here is a short youtube video that explains when to use "rem", "em" or even pixel values: https://www.youtube.com/watch?v=ZjiJIIL60GI